runtime/manta/src/migrations/assets_genesis.rs

```diff
@@ -0,0 +1,397 @@
+            return T::DbWeight::get().reads(2);
+        }
+
+        log::info!(target: "asset-manager", "Starting migration for AssetManager...");
+
+        let pallet_prefix: &[u8] = b"AssetManager";
+        let _ = clear_storage_prefix(pallet_prefix, b"AssetIdLocation", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"LocationAssetId", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"AssetIdMetadata", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"UnitsPerSecond", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"MinXcmFee", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"AllowedDestParaIds", b"", None, None);
+
+        put_storage_value(
+            pallet_prefix,
+            b"NextAssetId",
+            b"",
+            <T::AssetConfig as AssetConfig<T>>::StartNonNativeAssetId::get(),
+        );
+        let asset_id = <T::AssetConfig as AssetConfig<T>>::NativeAssetId::get();
+        let metadata = <T::AssetConfig as AssetConfig<T>>::NativeAssetMetadata::get();
+        let location = <T::AssetConfig as AssetConfig<T>>::NativeAssetLocation::get();
+        put_storage_value(
+            pallet_prefix,
+            b"AssetIdLocation",
+            &Blake2_128Concat::hash(&asset_id.encode()),
+            location.clone(),
+        );
+        put_storage_value(
+            pallet_prefix,
+            b"LocationAssetId",
+            &Blake2_128Concat::hash(&location.encode()),
+            asset_id,
+        );
+        put_storage_value(
+            pallet_prefix,
+            b"AssetIdMetadata",
+            &Blake2_128Concat::hash(&asset_id.encode()),
+            metadata,
+        );
+
+        StorageVersion::new(INITIAL_PALLET_ASSETS_MANAGER_VERSION + 1)
+            .put::<pallet_asset_manager::Pallet<T>>();
+
+        log::info!(target: "asset-manager", "✅ Storage migration for AssetManager has been executed successfully and storage version has been update to: {:?}.", INITIAL_PALLET_ASSETS_MANAGER_VERSION + 1);
+
+        log::info!(target: "assets", "Starting migration for pallet-assets...");
+
+        let pallet_prefix: &[u8] = b"Assets";
+        let _ = clear_storage_prefix(pallet_prefix, b"Asset", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"Metadata", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"Account", b"", None, None);
+        let _ = clear_storage_prefix(pallet_prefix, b"Approval", b"", None, None);
+
+        log::info!(target: "assets", "✅ Storage migration for Assets has been executed successfully and storage version has been update to: {:?}.", INITIAL_PALLET_ASSETS_MANAGER_VERSION + 1);
+
+        StorageVersion::new(INITIAL_PALLET_ASSETS_VERSION + 1).put::<pallet_assets::Pallet<T>>();
+
```

Comment:
Why not register the DOT and USDT assets while you're at it? Saves us another multisig coordination heachache

Reply:
I want to keep the migration work low, to review easier and also to not spend more dev time.
Was thinking the mutli-sig would be easier.